fails:Kernel#exit! exits with the given status
fails:Kernel#exit! exits immediately when called from a thread
fails:Kernel.exit! exits with the given status
fails:Kernel.exit! exits immediately when called from a thread
